src/pages/about.tsx

```diff
@@ -50,4 +54,17 @@ const SecondPage = ({ location }: ILocation) => {
   );
 };
 
+export const query = graphql`
```

Comment:
I repeat this query everywhere, need to test if I can create this in one place and then import to all pages

Reply:
I don't _think_ you can --- this is like the place for a fragment, which is not nearly as good as an import

Reply:
I guess you could also pass this in to every page as context, to avoid the duplication